test/plugin/base_test.rb

```diff
@@ -1424,6 +1454,81 @@ def test_cascading_json_detection_with_log_entry_trace_field
       nested_level_value: TRACE2)
   end
 
+  # Expected order.
+  # 1. Labels from the field "logging.googleapis.com/labels" in payload.
+  # 2. Labels from the config "label_map".
+  # 3. Labels from the config "labels".
+  def test_labels_order
```

Comment:
It's not clear to me what this is testing. Is this a priority order for labels provided with different mechanisms, that are all passed on in the same field? If so, I'd be more direct:
Verify that labels present in multiple inputs respect the expected priority order:
<then your list>

Reply:
Yup, that's exactly what the test is trying to verify. I've changed the test description accordingly.